patches/chrome-browser-ui-startup-startup_browser_creator.cc.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/chrome/browser/ui/startup/startup_browser_creator.cc b/chrome/browser/ui/startup/startup_browser_creator.cc
+index d58b84e43f35018aecdc7ddb9aa844f56b55d698..3836781ab7166f73e7d773c579f6e289fe6fec88 100644
+--- a/chrome/browser/ui/startup/startup_browser_creator.cc
++++ b/chrome/browser/ui/startup/startup_browser_creator.cc
+@@ -480,6 +480,9 @@ bool StartupBrowserCreator::InSynchronousProfileLaunch() {
+ Profile* StartupBrowserCreator::GetPrivateProfileIfRequested(
+     const base::CommandLine& command_line,
+     Profile* profile) {
++
```

Comment:
I took a stab at this in https://github.com/brave/brave-core/pull/8612/commits/0ff2eb6a9285fc3b5560ecac58c3884d72e02a90 but couldn't make it work because I don't understand the preprocessor magic well enough. Can you help with this, @darkdh?